### ISSUE TITLE
[Accessibility] Slider: Cannot Access in Scan Mode

### ISSUE THREAD

**Issue by HalcyonPride** [OfficeDev/office-ui-fabric-react#5628]
### Bug Report 
 
- __Package version(s)__: Latest, on https://developer.microsoft.com/en-us/fabric
- __Browser and OS versions__: Edge/Narrator, on Windows 10 Enterprise; Version 1803; OS 17134.167
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: **High**
 
Products/sites affected: All
 
#### Describe the issue:
 
Users cannot access Sliders in Scan Mode on Narrator.
- Turn on Scan Mode in Narrator.
- Go to https://developer.microsoft.com/en-us/fabric#/components/slider
- Using the arrow keys, in Scan Mode, navigate to the Sliders in the examples.
 
#### Actual behavior:
 
Focus skips over the Sliders, so users cannot access them.
 
#### Expected behavior:
 
Focus lands on the Sliders.
 
### If applicable, please provide a codepen repro:
 
Can repro here: https://developer.microsoft.com/en-us/fabric#/components/slider

**Comment by philipzloh** [comment]
Narrator bug?

**Comment by KevinTCoughlin** [comment]
Seems it since I can get focus on the Slider via keyboard and use the arrow keys to adjust input

**Comment by HalcyonPride** [comment]
Filed [this Bug](https://microsoft.visualstudio.com/OS/ft_xamlfoun/_workitems/edit/18584019) - we'll await the Narrator team's investigation

**Comment by HalcyonPride** [comment]
Narrator team has acknowledged the Bug.
